test/js/pages.test.js

```diff
@@ -100,7 +100,7 @@ describe('page filters', () => {
 
   test('eventSort', () => {
     const slugs = ['test3', 'events', 'test1'];
-    const years = [2018, 2019, 2020];
+    const years = ['2018', '2019', '2020'];
```

Comment:
I don't _think_ this causes any issues? They're now always strings, not numbers.